content/en/account_management/org_settings.md

```diff
@@ -88,7 +88,11 @@ The [**Safety Center**][14] page contains security alerts, warnings, and recomme
 
 ### Public sharing
 
-The **Public Sharing** tab contains lists of shared dashboards and shared graphs. You can also edit your sharing settings by clicking the **Enabled** toggles.
+The **Public Sharing** tab includes org-wide settings for sharing, along with lists of shared dashboards and graphs. You can enable sharing features granularly and configure additional security options, such as setting a maximum invite duration.
+
+If you require sharing settings to apply accross every org you own, please reach out to Datadog Support and we can set that up for you.
```

Comment:
```suggestion
To apply sharing settings across all your orgs, reach out to [Datadog Support][16].
```

Reply:
In addition to the edit above, could you please add the following reference link after line `177`? I'm unable to make a suggestion in that part of the page.

`[16]: /help/`